proposals/bulk-memory-operations/Overview.md

```diff
@@ -271,6 +271,17 @@ A trap occurs if:
 Note that it is allowed to use `memory.init` on the same data segment more than
 once.
 
+Initialization takes place bytewise from lower addresses toward higher
+addresses.  A trap resulting from an access outside the source data
+segment or target memory only occurs once the first byte that is
+outside the source or target is reached.  Bytes written before the
+trap stay written.
+
+Note that the semantics require bytewise accesses, so a trap that
```

Comment:
I think I know what you mean here, but the text is still a little unclear. Perhaps mention that an implementation may want to optimize this by copying words instead of bytes?

Reply:
Added some prose.